github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+# Ticket https://tignum.atlassian.net/browse/WEBDEV
```

Comment:
Please use the template we agreed upon https://github.com/TIGNUM/qot-web-application/blob/develop/.github/pull_request_template.md